context/config.go

```diff
@@ -64,8 +71,13 @@ func (c *SMFContext) insertSmfNssaiInfo(snssaiInfoConfig *factory.SnssaiInfoItem
 func (c *SMFContext) updateSmfNssaiInfo(modSliceInfo *factory.SnssaiInfoItem) error {
 	//identify slices to be updated
 	logger.InitLog.Infof("Network Slices to be modified [%v] ", factory.PrettyPrintNetworkSlices([]factory.SnssaiInfoItem{*modSliceInfo}))
-	c.deleteSmfNssaiInfo(modSliceInfo)
-	c.insertSmfNssaiInfo(modSliceInfo)
+	if err := c.deleteSmfNssaiInfo(modSliceInfo); err != nil {
```

Comment:
Will there be call failure if  Slice update comes and you are handling calls as well ?

Reply:
Yes, it is expected that no call processing is done when configuration update is done. Enhancement can be done to block calls until config update procedure is ongoing